ibis/backends/sql/compilers/datafusion.py

```diff
@@ -533,5 +532,8 @@ def visit_ArrayFlatten(self, op, *, arg):
     def visit_RandomUUID(self, op, **kw):
         return self.f.anon.uuid()
 
+    def visit_ArrayDistinct(self, op, *, arg):
+        return self.f.array_distinct(arg)
```

Comment:
This should be doable with a `SIMPLE_OPS` mapping.

Reply:
It certainly was, thank you for the suggestion!